src/compositor/compositor.c

```diff
@@ -257,6 +258,7 @@ deactivate_tty(struct wlc_compositor *compositor)
 
    compositor->state.tty = IDLE;
 
+   WLC_INTERFACE_EMIT(compositor.deactivated);
```

Comment:
Minor nitpick, generally code does not hug {} or if clauses in wlc, you can just put this under the above expression .. state.tty = IDLE; ..